### PR TITLE
Bump core-graphics dependency and version

### PR DESCRIPTION
**Pull request by glennw**
This change is [Reviewable](https://reviewable.io/reviews/servo/cocoa-rs/171)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name = "cocoa"
 description = "Bindings to Cocoa for OS X"
 homepage = "https://github.com/servo/cocoa-rs"
 repository = "https://github.com/servo/cocoa-rs"
-version = "0.9.2"
+version = "1.0.0"
```
Comment (nox) [inline Cargo.toml]: `0.10.0` please, this crate is nowhere near `1.0` quality, unfortunately. :(

**Comment by glennw** [comment]
@nox Fixed.

**Pull request merged**